<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toolbar_ramp_default_values</class>
 <widget class="QWidget" name="toolbar_ramp_default_values">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_min">
     <property name="text">
      <string>Default Min</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleEdit" name="lineEdit_min"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_max">
     <property name="text">
      <string>Default Max</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleEdit" name="lineEdit_max"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_steps">
     <property name="text">
      <string>Default Steps</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="spinBox_steps">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>240</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>qdoubleedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../openscantools.qrc"/>
 </resources>
 <connections/>
</ui>
